packages/errors/src/errors.ts

```diff
@@ -1342,19 +1342,23 @@ export const AllCypressErrors = {
   },
 
   CONFIG_FILE_DEV_SERVER_IS_NOT_A_FUNCTION: (configFilePath: string, setupNodeEvents: any) => {
```

Comment:
We should change the name of this error. We throw it based on if the `devServer` is not an object _or_ a function.

Reply:
Good catch... I made this more generic - `CONFIG_FILE_DEV_SERVER_IS_NOT_VALID`.

https://github.com/cypress-io/cypress/pull/21181/commits/b8f1f353b35ec6c2123917177ef69d5ade6ad4da